pkg/api/unversioned/group_version.go

```diff
@@ -48,9 +48,18 @@ type GroupVersion struct {
 	Version string
 }
 
+func (gv GroupVersion) IsEmpty() bool {
+	return len(gv.Group) == 0 && len(gv.Version) == 0
+}
+
 // String puts "group" and "version" into a single "group/version" string. For the legacy v1
 // it returns "v1".
 func (gv GroupVersion) String() string {
+	// special case the internal apiVersion for kube
```

Comment:
So the internal version of other groups will be "some-group/"? Why not making them "some-group", so it's consistent with using "" for "v1"?
I'm sorry if this is already discussed somewhere else.